AnnoDesigner/ViewModels/MainViewModel.cs

```diff
@@ -1447,11 +1287,27 @@ private void ExecutePlaceBuilding(object param)
 
         private void ExecuteShowPreferencesWindow(object param)
         {
-            var preferencesWindow = new PreferencesWindow(_appSettings, _commons, HotkeyCommandManager)
+            var preferencesWindow = new PreferencesWindow()
             {
                 Owner = Application.Current.MainWindow,
                 WindowStartupLocation = WindowStartupLocation.CenterOwner
             };
+            var vm = new PreferencesViewModel();
+            preferencesWindow.DataContext = vm;
+
+            vm.Pages.Add(new PreferencePage
+            {
+                Name = nameof(ManageKeybindingsPage),
+                ViewModel = PreferencesKeyBindingsViewModel,
+                Header = Localization.Localization.Translations["ManageKeybindings"]
```

Comment:
Replace with just translation key and translate using DynamicLocalize in XAML

Reply:
I don't understand it. What do I have to do?

Reply:
There is a new prepared binding in XAML, so you can replace `Header="{Binding Header}"` with `Header="{l:DynamicLocalize Header}"` and store just string "ManageKeybindings" in the Header property, the value of Header property will get translated with the binding.

Reply:
Personally I don't see the benefit.
Solution A has the "default" way of Databinding and the developer sees in XAML and C# where every data comes from.
Whereas in solution B the developer in C# has no idea that this string is a localization key and will later be used in XAML as a key for a translation converter/binding.

I always prefered solution A, because it is more expressive and self explanatory.
If solution B is the way to go in the future, I will adjust my PR.
What is @Agmas thinking?

Reply:
One benefit of solution B: it updates the rendered translation based on selected language.

Reply:
Well my solution B is also updating.
I can't check now why, but it does.

Reply:
If this is working at the moment as is, then I think @FroggieFrog is correct on this one. I can think of some locations where this dynamic binding will be useful, but I do agree that in this case, the meaning that is lost will just cause confusion.

Reply:
1. Open preference window
2. Change language without closing the preference window
![language](https://user-images.githubusercontent.com/7888949/87267880-2f582d80-c4c9-11ea-985a-edd8d5886863.png)

Also my language menu is SUPER big (the icons take up entire screen), but don't have time to check it out right now.

Reply:
I did a clean checkout and indeed the localization did not update. I don't know why it worked before.
But after implementing the `DynamicLocalize` the designer isn't useful anymore:
![localization_error](https://user-images.githubusercontent.com/6222752/87275330-24f45e80-c4de-11ea-8ce9-f20ec4f88114.png)

Also the I can't reproduce the issue with the images inside the language menu:
![languages](https://user-images.githubusercontent.com/6222752/87275472-6be25400-c4de-11ea-8f55-66bbd59a48e0.png)